.github/images/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:24.04 AS builder
+
+LABEL Description="Custom RSK node image to execute Rootstock Integration Tests"
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    wget gnupg2 curl git ca-certificates mocha  \
+    && apt clean
+
+# -- nodeJs ---------------------------------------------------------
+ENV NODE_VERSION  v20.14.0
+RUN mkdir -p /usr/local/nvm
+ENV NVM_DIR /usr/local/nvm
+
+RUN  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash \
+     && . $NVM_DIR/nvm.sh \
+     &&  nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/$NODE_VERSION/bin:$PATH
+
+# -- java ---------------------------------------------------------
+ENV JAVA_VERSION 11
+
+RUN apt-get  -y install "openjdk-$JAVA_VERSION-jdk"
+
+ENV JAVA_HOME="/usr/lib/jvm/java-$JAVA_VERSION-openjdk-amd64" \
+    PATH="$JAVA_HOME:$PATH"
+
+RUN export JAVA_HOME=$JAVA_HOME  \
+    && export PATH="$JAVA_HOME:$PATH"
+
+# -- bitcoind ---------------------------------------------------------
+ENV BITCOIN_VERSION 0.18.1
```

Comment:
We're running Bitcoin 24 in prod. Do we want 0.18.1 in here?

Reply:
@lucasvuotto , unfortunately, yes. Bitcoind 24 doesn't allow us certain flexibilities easily, like programmatically creating a wallet to spend from, etc.
I remember working on an story to figure out if we could use Bitcoind 24 for the rits and we were unable to get it.
We should probably analyze that later again.
We might have the result of that analysis somewhere. I will get back to you if I get it.

Reply:
Understood. Let's create a ticket for it and settle. :)

Reply:
@jeremy-then can you share me that story? I'm willing to help with it.